examples/js/effects/VREffect.js

```diff
@@ -307,11 +306,13 @@ THREE.VREffect = function( renderer, onError ) {
 			if ( camera.parent === null ) camera.updateMatrixWorld();
 
 			camera.matrixWorld.decompose( cameraL.position, cameraL.quaternion, cameraL.scale );
-			camera.matrixWorld.decompose( cameraR.position, cameraR.quaternion, cameraR.scale );
+			
+			cameraR.position.copy(cameraL.position);
```

Comment:
Yep :) `copy` is much cheaper than calculate again `decompose` and you just need the same values.